release notes/v0.39.0.md

```diff
@@ -0,0 +1,60 @@
+k6 v0.39.0 is here! :tada: It's a small release that includes a bunch of bugfixes and minor enhancements. A lot of our focus was on some upcoming big changes. You can read about what's coming up next at the end of this document.
```

Comment:
At the "end of the document" makes it seem like it will be at the end ... but it's actually from before the middle.  So we might want to rephrase it here or the head section to be "At the end starts here"

Reply:
As an alternative, we could directly mention the specific section.

```suggestion
k6 v0.39.0 is here! :tada: It's a small release that includes a bunch of bugfixes and minor enhancements. A lot of our focus was on some upcoming big changes. You can read about what's coming up next in the [Roadmap and future plan section](#roadmap-and-future-plans).
```

Reply:
hmm true :sweat_smile: how about something like this?

> A lot of our focus was on some upcoming big changes, described in the second part of this document

Reply:
Resolved in the [latest force-push](https://github.com/grafana/k6/compare/6f763113a21481f8b3d72ac5496366cd5ffbd982..3ab85778f61ebfdea240957dc995f084a466b270) with minor modifications.